packages/telemetry/src/telemetryReporter.ts

```diff
@@ -196,21 +203,24 @@ export class SpawnedTelemetryReporter extends AsyncCreatable<TelemetryOptions> {
 
 /**
  * Determine if the telemetry event should be logged.
- * Setting SFDX_DISABLE_INSIGHTS to true will disable insights for errors and diagnostics.
+ * Setting the disableTelemetry config var to true will disable insights for errors and diagnostics.
  */
-function isSfdxTelemetryEnabled(env: Env): boolean {
-  const sfdxDisableInsights = env.getBoolean(SFDX_DISABLE_INSIGHTS);
+function isSfdxTelemetryEnabled(config: ConfigAggregator): boolean {
+  const sfdxDisableInsights = config.getPropertyValue(DISABLE_TELEMETRY);
   const isEnabled = !sfdxDisableInsights;
   return isEnabled;
 }
 
-function logTelemetryStatus(env: Env, logger: Logger): void {
-  const isEnabled = isSfdxTelemetryEnabled(env);
-  logger.debug(`'${SFDX_DISABLE_INSIGHTS}': ${!isEnabled}`);
+function logTelemetryStatus(logger: Logger, config: ConfigAggregator): void {
+  const isEnabled = isSfdxTelemetryEnabled(config);
   if (isEnabled) {
-    logger.warn(`Insights logging in enabled. This can be disabled by setting ${SFDX_DISABLE_INSIGHTS}=true`);
+    logger.warn(
+      `Insights logging is enabled. This can be disabled by running sfdx force:config:set ${DISABLE_TELEMETRY}=true`
```

Comment:
Should we just day "Telemetry is enabled...". e.g. do they need to know it is insights?